src/main/scala/io/iohk/ethereum/blockchain/sync/PeersClient.scala

```diff
@@ -110,6 +98,24 @@ class PeersClient(
       case _: GetBlockBodies => BlockBodies.code
       case _: GetNodeData => NodeData.code
     }
+
+  private def printStatus(requesters: Requesters): Unit = {
+    log.debug(
+      "Request status: requests in progress: {}, available peers: {}",
+      requesters.size,
+      peersToDownloadFrom.size
+    )
+
+    val handshakedPeersStatus = handshakedPeers.map { case (peer, info) =>
```

Comment:
Minor: should be lazy to avoid evaluation when we work with `INFO` level of logging